static/app/components/preprod/preprodBuildsTable.tsx

```diff
@@ -110,6 +110,16 @@ export function PreprodBuildsTable({
                 <Text size="sm" variant="muted" monospace>
                   {(build.vcs_info?.head_sha?.slice(0, 7) || '--').toUpperCase()}
                 </Text>
+                {build.vcs_info?.pr_number !== null && (
+                  <React.Fragment>
+                    <Text size="sm" variant="muted">
+                      –
```

Comment:
```suggestion
                      #
```
wondering if using the hash sign is more appropriate for PRs?